src/app/shared/components/template/components/text-bubble/text-bubble.component.scss

```diff
@@ -74,80 +119,52 @@ $bubble-border-color-4: var(--text-bubble-border-color-4, --ion-color-secondary-
 // This creates the bubble shape, including the tail
 .text-bubble {
   /* triangle dimension */
-  --a: 75deg; /* angle */
-  --h: 1em; /* height */
+  --a: 0deg; /* angle */
+  --h: 0em; /* height */
 
-  --p: 50%; /* triangle position (0%:left 100%:right) */
-  &[data-position="left"] {
-    --p: calc(0% + #{$imageSize} + 12px);
+  --p: 0%; /* triangle position (0%:left 100%:right) */
+  position: relative;
+
+  clip-path: none;
+  border-image: none;
+  border-radius: 0px 12px 12px 12px;
+  border: 1px solid var(--ion-color-primary-700);
+  background-color: var(--ion-color-primary-50);
```

Comment:
nit(blocking): These two colours should be `$bubble-border-color-1` and `$bubble-background-color-1`, to preserve the current functionality where the default bubble colour is grey for the `default` theme (see [web preview](https://idems-debug.web.app/template/comp_text_bubble)).

Or, if you think that the primary colour makes sense as a better default colour, then the rules here should use `$bubble-border-color-2` and `$bubble-background-color-2`.